repondeur/tests/testapp.py

```diff
@@ -12,6 +12,13 @@ class TestResponse(BaseTestResponse):
     def parser(self):
         return HTMLParser(self.text)
 
+    @property
+    def title(self) -> str:
+        return self.parser.css_first("h1").text()
+
+    def __contains__(self, amendement) -> bool:
+        return bool(self.parser.css_first(f"#amendement-{amendement.num:06}"))
```

Comment:
Love the idea too. However when reading the test I was not sure what was happening exactly. Was the amendement being stringified maybe?

And what if we want to test the presence of other kinds of things on other pages? Maybe we could use a more explicit method name like contains_amendement?